### ISSUE TITLE
Documentation: New linked resources interface

### ISSUE THREAD

**Issue by jimsafley** [omeka/omeka-s#1964]
There's a new interface for linked resources that needs to be documented. This applies to a resource's "Linked resources" tab in the admin interface and a resource's "Linked resources" block in the public interface. Before, it listed the available properties a user can filter, now users can filter by the available properties *and* resource type (item, item set, & media). See #1962 for instructions on how to set up linked resources for this purpose.

**Comment by allanaaa** [comment]
This is only supposed to work in the default theme, right? (After editing the version constraint bit.)

**Comment by jimsafley** [comment]
Right now we're transitioning themes and modules to be compatible with the new major version (4.0.0), so it will work on every theme, eventually.

**Comment by allanaaa** [comment]
Yes, sorry, I meant for testing purposes right now.

**Comment by jimsafley** [comment]
Yes, the first four reflect expected behavior. As for the last one, are you certain you don't see a "Media" group in the linked resources dropdown menu?

**Comment by allanaaa** [comment]
Ohhhhhhhhhhhhh! It's not at all clear that the Linked resources table is just loading Items by default. When "All" shows up in the dropdown box, that should really be ... something else. 

Ideally, the table would show all resources by default, then you can narrow by resource type later. If you're not keen on that, then the dropdown should show "Item: All" and "Item: [Property]" as helper text instead. I would also change the section title to "Linked Resources: [Resource Type]". Having it show up lower just makes it looks like Items is all there is.

**Comment by jimsafley** [comment]
We cannot have an "All resources" option due to the incompatibility of queries needed to filter resources of different types. It has to be one resource type at a time.

I've committed changes that should address the problem (0fbc0a7913212778e9ddbcd2a1c9b492d927a732). I think we should leave the section title as-is. @kimisgold Do you have any thoughts on changing the "Linked resources" section title to include whatever resource type is active?

**Comment by kimisgold** [comment]
I'm also fine with leaving "Linked Resources" as is for the section title, especially admin side where we don't want to have super long strings for the tabs. I'll do a pass on styles to see if I can make the current selected resource a little more obvious.
